lib/src/app.dart

```diff
@@ -18,9 +18,9 @@
 import 'dart:io';
 
 import 'package:meta/meta.dart';
+import 'package:path/path.dart' as _path;
```

Comment:
I think the linter will complain that you prefix with _ when naming an imported package.

Reply:
no warnings exists

Reply:
info - src/app.dart:21:36 - Avoid leading underscores for library prefixes. - no_leading_underscores_for_library_prefixes

Reply:
https://dart.dev/guides/language/effective-dart/style#dont-use-a-leading-underscore-for-identifiers-that-arent-private

Reply:
I don't see a warning in the editor

Reply:
Also I think we were already using this naming elsewhere